util/util.go

```diff
@@ -317,4 +323,23 @@ func init() {
 	oneLineJSONCodecHandle.HTMLCharsAsIs = true
 	oneLineJSONCodecHandle.Indent = 0
 	oneLineJSONCodecHandle.MapKeyAsString = true
+
+	prettyHandle = new(codec.JsonHandle)
+	prettyHandle.ErrorIfNoField = json.CodecHandle.ErrorIfNoField
+	prettyHandle.ErrorIfNoArrayExpand = json.CodecHandle.ErrorIfNoArrayExpand
+	prettyHandle.Canonical = json.CodecHandle.Canonical
+	prettyHandle.RecursiveEmptyCheck = json.CodecHandle.RecursiveEmptyCheck
+	prettyHandle.Indent = json.CodecHandle.Indent
+	prettyHandle.HTMLCharsAsIs = json.CodecHandle.HTMLCharsAsIs
+	prettyHandle.MapKeyAsString = true
+	prettyHandle.Indent = 2
+
+	jsonStrictHandle = new(codec.JsonHandle)
+	jsonStrictHandle.ErrorIfNoField = prettyHandle.ErrorIfNoField
+	jsonStrictHandle.ErrorIfNoArrayExpand = prettyHandle.ErrorIfNoArrayExpand
+	jsonStrictHandle.Canonical = prettyHandle.Canonical
+	jsonStrictHandle.RecursiveEmptyCheck = prettyHandle.RecursiveEmptyCheck
+	jsonStrictHandle.Indent = prettyHandle.Indent
+	jsonStrictHandle.HTMLCharsAsIs = prettyHandle.HTMLCharsAsIs
+	jsonStrictHandle.MapKeyAsString = true
```

Comment:
trying to copy the protocol.JSONStrictHandle. https://github.com/algorand/go-algorand/blob/master/protocol/codec.go#L41-L43. 
In the case of the indexer, jsonStrictHandle should be the same as prettyHandle, except for the MapKeyAsString config?

Reply:
Yes I think thats right. :+1: